.yamato/mobile-build-and-test.yml

```diff
@@ -0,0 +1,93 @@
+2021.2_Build_Android_player:
+  name: 2021.2 Build Android player
+  agent:
+      type: Unity::VM
+      image: package-ci/win10:stable
+      flavor: b1.large
+  commands:
+    - pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+    - curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr.bat --output utr.bat
+    - | 
+       mkdir C:\TMP
```

Comment:
@andrews-unity had pointed this weirdness out. The issue goes like this: Android is built on Windows, Windows has path length issues so the default Editor location results in unreadable paths. The suggestion was "don't use Windows" and when I explored Mac and Linux I got feedback from various folks to not use Mac as it was a scarce resource and not to use Linux as most configs for android are set up on Windows and Linux hasn't been set up so there will probably be the need for a custom bokken image for that. So I'm back to the TMP dir on Windows. 
I'm not familiar with how to use macros to circumvent this, but it would be great if we could. I just ran around in circles for a while and gave up. @JesseOlmer

Reply:
I also mentioned to using the mobile image :P 

https://github.cds.internal.unity3d.com/unity/com.unity.transport/blob/master/.yamato/mobile-playmode-tests.yml#L6

This is how we are doing it on UTP and we use a windows image to build for android. This is also the same setup that dots is using to build their android player and their folder depth is way longer than ours.